qiskit/visualization/dag_visualization.py

```diff
@@ -31,6 +31,63 @@
 except ImportError:
     HAS_PIL = False
 
+FILENAME_EXTENSIONS = [
+    "bmp",
+    "canon",
+    "cgimage",
+    "cmap",
+    "cmapx",
+    "cmapx_np",
```

Comment:
Where did you find this full list?  Perhaps we could cut it down to some more common ones - I think things like `cmapx_np` might be a bit more noise than use.

Reply:
pydot library was throwing a exception with all those extensions...

Reply:
Ah, I see - it's the failing output of `dot -Tunknown`.